python/ml4ir/base/features/feature_fns/tf_native.py

```diff
@@ -0,0 +1,59 @@
+import tensorflow as tf
+
+from ml4ir.base.io.file_io import FileIO
+
+
+def tf_native_op(feature_tensor: tf.Tensor, feature_info: dict, file_io: FileIO):
```

Comment:
This is the main addition of this PR.

Reply:
Do we have to also add a short paragraph on the public doc (https://ml4ir.readthedocs.io/en/latest/) about this new capability?

Reply:
Good call. Let me do that.

Reply:
Good call. Let me do that.